Sources/PaymentsCore/Models/OrderData.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+public struct OrderData {
+    let orderID: String
+    let status: OrderStatus?
```

Comment:
currently `OrderData` only has orderID and status, but the actual network response for `/confirm-payment-source` also includes `payment-source`:
```
{
    "id": "6NV18671YP407052Y",
    "status": "APPROVED",
    "payment_source": {
        "card": {
            "last_digits": "7321",
            "brand": "VISA",
            "type": "CREDIT"
        }
    }
}
```
Should this be addressed and included in this PR?
Or leave as is for simplicity's sake and we'll consider adding this when more payment methods are implemented?

Reply:
We came across this too. I believe we flattened the response into a Kotlin `data class` in a `CardResponse` object to decouple ourselves from the RESTful response's JSON schema.